lib/wise_homex/api_config.exs

```diff
@@ -153,6 +153,14 @@ endpoints = [
     path: "/foreign-devices",
     type: "foreign-devices"
   },
+  %{
+    endpoints: [:index, :show, :create, :update, :delete],
+    model: WiseHomex.FuelCheckPoint,
+    name_plural: "fuel_check_points",
+    name_singular: "fuel_check_point",
+    path: "/fuel-check-points",
+    type: "fuel-check-points"
+  },
```

Comment:
You also need to add the appropriate functions in the [behaviour](https://github.com/wise-home/wise_homex/blob/master/lib/wise_homex/api_client_behaviour.ex). That gives us compile-time guarantee that the different implementations are there.
Since the endpoints defined in this way are added automatically with meta programming, nothing else than adding it to the behaviour is needed.

Reply:
Fixed in b1e6ad5 :sunny: